extra_data/components.py

```diff
@@ -1625,6 +1740,7 @@ class JUNGFRAU(MultimodDetectorBase):
         r'(MODULE_|RECEIVER-|JNGFR)(?P<modno>\d+)'
     )
     _main_data_key = 'data.adc'
+    _mask_data_key = 'data.mask'
```

Comment:
This is probably not an urgent problem to block this PR, but I think this would not copy the mask's compression and could blow them up quite a bit. I suspect in the mask case, compression is always a performance benefit.

Reply:
Sorry, I'm a bit confused - where would the compression be copied to?

Reply:
`data.mask` is a product of offline corrections and generally stored compressed. If I'm not mistaken, this `FileWriter` will not maintain the compression and significantly blow up the data size.

Reply:
Ah, is this folded diff deceiving you?

![image](https://github.com/European-XFEL/EXtra-data/assets/327925/e27d3fb0-ab7b-4803-935b-17fd656fa6aa)

This line is in the JUNGFRAU class, not FramesFileWriter. I don't think this MR affects writing files at all.

Reply:
Uhm, what made GitHub skip so many class definitions?

In any case, ignore the comment, sorry.